egs2/ml_superb/asr1/local/data_prep.py

```diff
@@ -263,6 +265,17 @@ def process_text(text):
                 utt_id, _, text = line
                 if reserve_flag and utt_id not in FEW_SHOT_SELECTED_DATA[lang]:
                     continue
+
+                # skip wavefile over 20s
+                # the default setting in ml-superb benchmark (asr1)
+                wav_details, sample_rate = sf.read(
+                    os.path.join(
+                        args.source, dataset, lang, "wav", "{}.wav".format(utt_id)
+                    )
+                )
+                if len(wav_details) / sample_rate > 20:
```

Comment:
Can you make 20 seconds as a variable and 20 seconds as the default value?
Also, how about adding a warning to cut such data with the file ID, which will make it easy for the user to understand this thresholding process??